src/plugins/jartic/archive_downloader.py

```
@@ -0,0 +1,241 @@
+            raise
+
+    async def download_archives_range(
+        self,
+        start_year: int,
+        start_month: int,
+        end_year: int,
+        end_month: int
+    ) -> List[Path]:
+        downloaded_paths = []
+
+        current_date = datetime(start_year, start_month, 1)
```

Ruff (0.12.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)